public/js/monitor.js

```diff
@@ -62,6 +62,35 @@ function doOauth(el, {emailWatch = false} = {}) {
     }
   });
 
+  // Growth Experiment: OAuth Entry Point IDs are unique to the experiment.
+  const oAuthEntryPointIds = [
+    "fx-monitor-check-for-breaches-blue-btn",
+    "fx-monitor-find-out-blue-btn",
+    "fx-monitor-alert-me-blue-btn-top",
+    "fx-monitor-alert-me-blue-btn-bottom",
+  ];
+
+  if (oAuthEntryPointIds.includes(el.dataset.entrypoint)) {
+    // Growth Experiment: Reset UTMs from in-line body tag data elements.
+    ["utm_source", "utm_medium", "utm_campaign", "utm_term", "utm_content" ].forEach(key => {
```

Comment:
DRY: put this list of values into a constant and use it here and in `handleFormSubmits`